src/dev/flang/tools/docs/Html.java

```diff
@@ -182,9 +193,10 @@ private String summary(AbstractFeature af, boolean printArgs)
 
 
   private String anchor(AbstractFeature af) {
-    return "<a class='fd-feature font-weight-600' href='" + featureAbsoluteURL(af) + "'>"
+    return "<span class='font-weight-600'>" + typePrfx(af)
```

Comment:
General rule of thumb, use `div` instead of `span` if not 100% sure that all children are inline elements.

Reply:
Hm I thought I tried this and it caused problems, but it works now. I guess I had the type prefix outside the div before.